### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by misikira**
1. has a name input field
2. has email input
3. has the facility to send message 
4. data validation as per project requirement

### CODE REVIEW

**Comment by misikira** [comment]
microverse

**Review by amejid — COMMENTED** [review]
Hi @misikira, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a README file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @misikira :wave: , 

Good job with this milestone so far!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by misikira** [comment]
oh my god I accidentally closed and merged my intention was to merge the animation branch which doesn't need reviewer feedback I am working to resolve this

File: index.html [hunk 1]
```diff
@@ -237,6 +237,32 @@ <h1 class="head-About">About</h1>
...
                 
+                <div class="get-started">
+                    <img src="./img/header/header-logo.svg" alt="" class="get-started-logo">
+                    <h1 class="Get-started-head">Get started</h1>
+                    <p class="Get-started-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+                    <form action="https://formspree.io/f/xyyawqnj" class="form" method="post">
+                        <div class="form-item">
+                            <label for="name" class="your-name" >Your name</label><br>
+                            <input class="input-name" type="text" placeholder="Misikir" id="name"
+                             label="Your name" maxlength="15" name="name" >
```
Comment (Abenezer-Tilahun) [inline index.html:248]: - [ ] As stated in the project requirement Kindly make sure that **Name field accepts up to 30 chars.** and **A user cannot submit the form if one or more of the fields is empty.** So Kindly make `maxlength="30"` and add `required` attribute to the name field.

File: index.html [hunk 2]
```diff
@@ -237,6 +237,32 @@ <h1 class="head-About">About</h1>
...
+                        </div>
+                        <div class="form-item-email">
+                            <label for="email-feild" class="your-email">Your email</label><br>
+                            <input type="email" required placeholder="|" id="email-feild"
+                            class="form-input" label="Your email" name="email" name="your message">
```
Comment (Abenezer-Tilahun) [inline index.html:253]: - [ ] As stated in the project requirement Kindly make sure that **A user cannot submit the form if one or more of the fields is empty**. So Kindly make  add `required` attribute to the email field.

**Review by Abenezer-Tilahun — COMMENTED** [review]
Hi @misikira  , 

Great job for implemented  the suggested changes by the previous reviewer , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No Linters error
✔️ You used correct Git Hub-Flow.
✔️ Good Commit Messages
✔️ Professional READEME 


almost

## Required Changes ♻️
_Check the comments below the review._

- [x] For Your future project kindly make sure that before you got approved for this project activity kindly i recommend you not to merge to main branch.  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (misikira) [reply index.html:253]: @Abenezer-Tilahun  thank you for your helpful recommendations and fixed the issues as per your review the problem I am having is I accidentally merged and closed my branch if I push from this branch will it create a new pull request how can I resolve this

Comment (Abenezer-Tilahun) [reply index.html:253]: Hello @misikira I see that so implemented all the change and submit with new pull request and on the pull request description  please refer this pull request for the next reviewer reference.

Comment (misikira) [reply index.html:253]: Thanks for reaching me out I just did that got approved

Comment (misikira) [reply index.html:253]: @Abenezer-Tilahun  Thanks for reaching me I just did that and got approved

**Pull request merged**